src/Xamarin.Android.Build.Tasks/MSBuild/Xamarin/Android/Xamarin.Android.Bindings.Core.targets

```diff
@@ -128,21 +137,35 @@ It is shared between "legacy" binding projects and .NET 5 projects.
         Nullable="$(Nullable)"
         UseJavaLegacyResolver="$(_AndroidUseJavaLegacyResolver)"
         NamespaceTransforms="@(AndroidNamespaceReplacement)"
-    />
+        GeneratedFileListFile="$(GeneratedOutputPath)src\$(AssemblyName).projitems"
```

Comment:
Does `<Generator/>` already write this file? I was having trouble finding what creates it.